protocol/offsetdelete/offsetdelete_test.go

```diff
@@ -0,0 +1,29 @@
+package offsetdelete_test
+
+import (
+	"testing"
+
+	"github.com/segmentio/kafka-go/protocol/offsetdelete"
+	"github.com/segmentio/kafka-go/protocol/prototest"
+)
+
+func TestOffsetDeleteequest(t *testing.T) {
```

Comment:
Any reason not to add a similar test for the response type?

Reply:
Fixed in [0db9857](https://github.com/segmentio/kafka-go/pull/1010/commits/0db9857b3a9812cd4cff914400aff47b0501d164)